384 WARNING /usr/local/lib/python2.7/dist-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

384 WARNING /usr/local/lib/python2.7/dist-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)
738 WARNING /usr/local/lib/python2.7/dist-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

738 WARNING /usr/local/lib/python2.7/dist-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

774 ERROR Internal Server Error: /job_add/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/home/yes/yesw/git/lynn-2.0.0/src/web/lynn/service/views.py", line 70, in job_add
    timeout = form.cleaned_date["timeout"]
AttributeError: 'JobSubmit' object has no attribute 'cleaned_date'
774 ERROR Internal Server Error: /job_add/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/home/yes/yesw/git/lynn-2.0.0/src/web/lynn/service/views.py", line 70, in job_add
    timeout = form.cleaned_date["timeout"]
AttributeError: 'JobSubmit' object has no attribute 'cleaned_date'
934 WARNING /usr/local/lib/python2.7/dist-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

934 WARNING /usr/local/lib/python2.7/dist-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

968 ERROR Internal Server Error: /job_add/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/home/yes/yesw/git/lynn-2.0.0/src/web/lynn/service/views.py", line 85, in job_add
    s = string.replace(s, "T_TIME", timeout)
  File "/usr/lib/python2.7/string.py", line 519, in replace
    return s.replace(old, new, maxreplace)
TypeError: coercing to Unicode: need string or buffer, int found
968 ERROR Internal Server Error: /job_add/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/home/yes/yesw/git/lynn-2.0.0/src/web/lynn/service/views.py", line 85, in job_add
    s = string.replace(s, "T_TIME", timeout)
  File "/usr/lib/python2.7/string.py", line 519, in replace
    return s.replace(old, new, maxreplace)
TypeError: coercing to Unicode: need string or buffer, int found
225 ERROR Internal Server Error: /job_add/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/home/yes/yesw/git/lynn-2.0.0/src/web/lynn/service/views.py", line 85, in job_add
    s = string.replace(s, "T_TIME", timeout)
  File "/usr/lib/python2.7/string.py", line 519, in replace
    return s.replace(old, new, maxreplace)
TypeError: coercing to Unicode: need string or buffer, int found
225 ERROR Internal Server Error: /job_add/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/home/yes/yesw/git/lynn-2.0.0/src/web/lynn/service/views.py", line 85, in job_add
    s = string.replace(s, "T_TIME", timeout)
  File "/usr/lib/python2.7/string.py", line 519, in replace
    return s.replace(old, new, maxreplace)
TypeError: coercing to Unicode: need string or buffer, int found
215 ERROR Internal Server Error: /job_add/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/home/yes/yesw/git/lynn-2.0.0/src/web/lynn/service/views.py", line 85, in job_add
    str(timeout)
  File "/usr/lib/python2.7/string.py", line 519, in replace
    return s.replace(old, new, maxreplace)
TypeError: coercing to Unicode: need string or buffer, int found
215 ERROR Internal Server Error: /job_add/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/home/yes/yesw/git/lynn-2.0.0/src/web/lynn/service/views.py", line 85, in job_add
    str(timeout)
  File "/usr/lib/python2.7/string.py", line 519, in replace
    return s.replace(old, new, maxreplace)
TypeError: coercing to Unicode: need string or buffer, int found
066 WARNING /usr/local/lib/python2.7/dist-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

066 WARNING /usr/local/lib/python2.7/dist-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

2014-05-21 17:01:22,107 ERROR Internal Server Error: /job_add/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/home/yes/yesw/git/lynn-2.0.0/src/web/lynn/service/views.py", line 86, in job_add
    s = string.replace(s, "T_TIME", timeout)
  File "/usr/lib/python2.7/string.py", line 519, in replace
    return s.replace(old, new, maxreplace)
TypeError: coercing to Unicode: need string or buffer, int found
2014-05-21 17:01:22,107 ERROR Internal Server Error: /job_add/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/home/yes/yesw/git/lynn-2.0.0/src/web/lynn/service/views.py", line 86, in job_add
    s = string.replace(s, "T_TIME", timeout)
  File "/usr/lib/python2.7/string.py", line 519, in replace
    return s.replace(old, new, maxreplace)
TypeError: coercing to Unicode: need string or buffer, int found
2014-05-21 17:04:00,286 ERROR Internal Server Error: /job_add/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/home/yes/yesw/git/lynn-2.0.0/src/web/lynn/service/views.py", line 86, in job_add
    fo.write(s)
  File "/usr/lib/python2.7/string.py", line 519, in replace
    return s.replace(old, new, maxreplace)
TypeError: coercing to Unicode: need string or buffer, int found
2014-05-21 17:04:00,286 ERROR Internal Server Error: /job_add/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/home/yes/yesw/git/lynn-2.0.0/src/web/lynn/service/views.py", line 86, in job_add
    fo.write(s)
  File "/usr/lib/python2.7/string.py", line 519, in replace
    return s.replace(old, new, maxreplace)
TypeError: coercing to Unicode: need string or buffer, int found
2014-05-21 17:14:32,183 WARNING /usr/local/lib/python2.7/dist-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

2014-05-21 17:14:32,183 WARNING /usr/local/lib/python2.7/dist-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

2014-05-21 17:14:32,222 ERROR Internal Server Error: /job_add/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/home/yes/yesw/git/lynn-2.0.0/src/web/lynn/service/views.py", line 111, in job_add
    return HttpResponse("haha")
NameError: global name 'HttpResponse' is not defined
2014-05-21 17:14:32,222 ERROR Internal Server Error: /job_add/
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py", line 115, in get_response
    response = callback(request, *callback_args, **callback_kwargs)
  File "/home/yes/yesw/git/lynn-2.0.0/src/web/lynn/service/views.py", line 111, in job_add
    return HttpResponse("haha")
NameError: global name 'HttpResponse' is not defined
2014-05-21 19:42:26,085 WARNING /usr/local/lib/python2.7/dist-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

2014-05-21 19:42:26,085 WARNING /usr/local/lib/python2.7/dist-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead
  DeprecationWarning)

2014-05-21 19:43:13,011 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:43:13,011 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:43:13,012 ERROR set task info error
2014-05-21 19:43:13,012 ERROR set task info error
2014-05-21 19:43:14,867 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:43:14,867 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:43:14,868 ERROR set task info error
2014-05-21 19:43:14,868 ERROR set task info error
2014-05-21 19:44:31,155 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:44:31,155 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:44:31,155 ERROR set task info error
2014-05-21 19:44:31,155 ERROR set task info error
2014-05-21 19:44:33,011 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:44:33,011 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:44:33,011 ERROR set task info error
2014-05-21 19:44:33,011 ERROR set task info error
2014-05-21 19:45:49,299 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:45:49,299 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:45:49,299 ERROR set task info error
2014-05-21 19:45:49,299 ERROR set task info error
2014-05-21 19:45:51,155 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:45:51,155 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:45:51,155 ERROR set task info error
2014-05-21 19:45:51,155 ERROR set task info error
2014-05-21 19:47:07,443 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:47:07,443 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:47:07,443 ERROR set task info error
2014-05-21 19:47:07,443 ERROR set task info error
2014-05-21 19:47:09,299 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:47:09,299 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:47:09,299 ERROR set task info error
2014-05-21 19:47:09,299 ERROR set task info error
2014-05-21 19:48:25,587 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:48:25,587 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:48:25,587 ERROR set task info error
2014-05-21 19:48:25,587 ERROR set task info error
2014-05-21 19:48:27,443 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:48:27,443 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:48:27,443 ERROR set task info error
2014-05-21 19:48:27,443 ERROR set task info error
2014-05-21 19:49:43,731 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:49:43,731 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:49:43,731 ERROR set task info error
2014-05-21 19:49:43,731 ERROR set task info error
2014-05-21 19:49:45,587 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:49:45,587 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:49:45,587 ERROR set task info error
2014-05-21 19:49:45,587 ERROR set task info error
2014-05-21 19:51:01,875 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:51:01,875 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:51:01,875 ERROR set task info error
2014-05-21 19:51:01,875 ERROR set task info error
2014-05-21 19:51:03,731 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:51:03,731 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:51:03,731 ERROR set task info error
2014-05-21 19:51:03,731 ERROR set task info error
2014-05-21 19:52:20,019 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:52:20,019 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:52:20,019 ERROR set task info error
2014-05-21 19:52:20,019 ERROR set task info error
2014-05-21 19:52:21,875 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:52:21,875 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:52:21,875 ERROR set task info error
2014-05-21 19:52:21,875 ERROR set task info error
2014-05-21 19:53:38,163 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:53:38,163 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:53:38,163 ERROR set task info error
2014-05-21 19:53:38,163 ERROR set task info error
2014-05-21 19:53:40,019 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:53:40,019 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:53:40,019 ERROR set task info error
2014-05-21 19:53:40,019 ERROR set task info error
2014-05-21 19:54:56,307 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:54:56,307 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:54:56,307 ERROR set task info error
2014-05-21 19:54:56,307 ERROR set task info error
2014-05-21 19:54:58,163 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:54:58,163 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:54:58,163 ERROR set task info error
2014-05-21 19:54:58,163 ERROR set task info error
2014-05-21 19:56:14,451 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:56:14,451 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:56:14,452 ERROR set task info error
2014-05-21 19:56:14,452 ERROR set task info error
2014-05-21 19:56:16,307 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:56:16,307 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:56:16,307 ERROR set task info error
2014-05-21 19:56:16,307 ERROR set task info error
2014-05-21 19:57:32,595 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:57:32,595 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:57:32,595 ERROR set task info error
2014-05-21 19:57:32,595 ERROR set task info error
2014-05-21 19:57:34,451 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:57:34,451 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:57:34,451 ERROR set task info error
2014-05-21 19:57:34,451 ERROR set task info error
2014-05-21 19:58:50,739 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:58:50,739 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:58:50,739 ERROR set task info error
2014-05-21 19:58:50,739 ERROR set task info error
2014-05-21 19:58:52,595 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:58:52,595 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 19:58:52,595 ERROR set task info error
2014-05-21 19:58:52,595 ERROR set task info error
2014-05-21 20:00:08,883 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 20:00:08,883 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 20:00:08,883 ERROR set task info error
2014-05-21 20:00:08,883 ERROR set task info error
2014-05-21 20:00:10,739 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 20:00:10,739 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 20:00:10,739 ERROR set task info error
2014-05-21 20:00:10,739 ERROR set task info error
2014-05-21 20:01:27,027 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 20:01:27,027 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 20:01:27,027 ERROR set task info error
2014-05-21 20:01:27,027 ERROR set task info error
2014-05-21 20:01:28,883 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 20:01:28,883 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 20:01:28,883 ERROR set task info error
2014-05-21 20:01:28,883 ERROR set task info error
2014-05-21 20:02:45,171 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 20:02:45,171 ERROR Could not connect to 10.128.85.10:9999
2014-05-21 20:02:45,171 ERROR set task info error
171 ERROR set task info error
027 ERROR Could not connect to 10.128.85.10:9999
027 ERROR Could not connect to 10.128.85.10:9999
027 ERROR set task info error
027 ERROR set task info error
315 ERROR Could not connect to 10.128.85.10:9999
315 ERROR Could not connect to 10.128.85.10:9999
315 ERROR set task info error
315 ERROR set task info error
171 ERROR Could not connect to 10.128.85.10:9999
171 ERROR Could not connect to 10.128.85.10:9999
171 ERROR set task info error
171 ERROR set task info error
459 ERROR Could not connect to 10.128.85.10:9999
459 ERROR Could not connect to 10.128.85.10:9999
459 ERROR set task info error
459 ERROR set task info error
315 ERROR Could not connect to 10.128.85.10:9999
315 ERROR Could not connect to 10.128.85.10:9999
315 ERROR set task info error
315 ERROR set task info error
603 ERROR Could not connect to 10.128.85.10:9999
603 ERROR Could not connect to 10.128.85.10:9999
